cornavirus.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Novel Coronavirus 2020
```

Comment:
I think it is 2019 in the naming. I'd probably have had COVID-19 in the title too if i could